### PR TITLE
Home: Added Routes 

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
- Added pages: productpage, wishlist, cart
- Added routes to these pages

### CODE REVIEW

File: src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
 import "../Navbar/Navbar.css";
+import {Link} from "react-router-dom";
 
 export default function Navbar(){
     return(
         <header className="header">
         <div className="lilac-navbar">
             {/* logo */}
```
Comment (GeetaPogul) [inline src/components/Navbar/Navbar.js:8]: remove commented code

**Review by GeetaPogul — APPROVED** [review]
remove commented code 
looks good to me

File: src/components/Navbar/Navbar.js (hunk 1 above)
Comment (muskan-gupta-0803) [reply src/components/Navbar/Navbar.js:8]: Sure ,I will remove .
Thanks for reviewing.

File: src/App.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import "./App.css";
-
-import Homepage  from "./pages/Homepage/Homepage.js";
+import { Routes, Route } from "react-router-dom";
+// import Homepage  from "./pages/Homepage/Homepage.js";
+import {Homepage, ProductPage, Wishlist, Cart} from "./pages";
 
 import Navbar from "./components/Navbar/Navbar.js";
```
Comment (AtulHande03a) [inline src/App.js:6]: use absolute path to import components.. will be much more redable.

File: src/components/Navbar/Navbar.js [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-            <a className="logo" href="/index.html">
+            <Link to="/">
                 <div className="lilac-nav-logo nav-logo">
                     Style Store
                 </div>
-            </a>    
+            </Link>    
 
             {/*  nav search bar  */}
```
Comment (AtulHande03a) [inline src/components/Navbar/Navbar.js:15]: remove the commented code if possible

**Review by AtulHande03a — COMMENTED** [review]
rest all looks good!!

File: src/App.js (hunk 2 above)
Comment (muskan-gupta-0803) [reply src/App.js:6]: Sure I will add

File: src/components/Navbar/Navbar.js (hunk 3 above)
Comment (muskan-gupta-0803) [reply src/components/Navbar/Navbar.js:15]: Okay, thanks for reviewing

**Pull request merged**